core-web/libs/sdk/angular/README.md

```diff
@@ -23,111 +37,83 @@ Or using Yarn:
-```
-export const appConfig: ApplicationConfig = {
-    providers: [
-        provideDotcmsClient(DOTCMS_CLIENT_CONFIG),
-        provideRouter(routes),
-    ],
-};
-```
-## Resolver
-```javascript
-export const routes: Routes = [
-  {
-    path: '**',
-    resolve: {
-      // This should be called `context`.
-      context: DotCMSPageResolver,
-    },
-    component: DotCMSPagesComponent,
-    runGuardsAndResolvers: 'always' // Run the resolver on every navigation. Even if the URL hasn't changed.
-  },
-];
-```
+## DotCMS Page API
 
-Then, in your component, you can read the data using
+The `DotcmsLayoutComponent` requires a `DotCMSPageAsset` object to be passed in to it. This object represents a dotCMS page and can be fetched using the `@dotcms/client` library.
 
-```javascript
-protected readonly context = signal(null);
+- [DotCMS Official Angular Example](https://github.com/dotCMS/core/tree/master/examples/angular)
+- [`@dotcms/client` documentation](https://www.npmjs.com/package/@dotcms/client)
+- [Page API documentation](https://dotcms.com/docs/latest/page-api)
 
-ngOnInit() {
-    // Get the context data from the route
-    this.route.data.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(data => {
-        this.context.set(data['context']);
-    });
-}
-```
 ## Components
 
-### `DotcmsLayoutComponent`
+### DotcmsLayoutComponent
 
-A component that renders a layout for a dotCMS page.
+The `DotcmsLayoutComponent` is a crucial component for rendering dotCMS page layouts in your Angular application.
 
 #### Inputs
 
--   **entity**: The context for a dotCMS page.
--   **components**: An object with the relation of contentlets and the component to render each.
-
-
-#### Usage
-
-```javascript
-    <dotcms-layout [entity]="pageAsset" [components]="components()" />
-
-    DYNAMIC_COMPONENTS: { [key: string]: DynamicComponentEntity } = {
-    Activity: import('../pages/content-types/activity/activity.component').then(
-        (c) => c.ActivityComponent,
-    ),
-    Banner: import('../pages/content-types/banner/banner.component').then(
-        (c) => c.BannerComponent,
-    ),
-    Image: import('../pages/content-types/image/image.component').then(
-        (c) => c.ImageComponent,
-    ),
-    webPageContent: import(
-        '../pages/content-types/web-page-content/web-page-content.component'
-    ).then((c) => c.WebPageContentComponent),
-    Product: import('../pages/content-types/product/product.component').then(
-        (c) => c.ProductComponent,
-    ),
+| Name         | Type                 | Description                                                           |
+|--------------|----------------------|-----------------------------------------------------------------------|
+| `pageAsset`  | `DotCMSPageAsset`    | The object representing a dotCMS page from PageAPI response.          |
+| `components` | `DotCMSPageComponent`| An object mapping contentlets to their respective render components.  |
```

Comment:
This type is wrong.

Reply:
We have this type and I remember changing it in the lib in this PR, but the change is not there, I already put it again